src/Style/style.css

```diff
@@ -0,0 +1,207 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #d7d7d7;
+  font-family: sans-serif;
+}
+
+.to-do-wrapper {
+  margin: 10rem;
+  background-color: #fff;
+}
+
+.input-label {
+  display: flex;
+  gap: 1rem;
+}
+
+h3 {
+  padding-left: 1rem;
+  padding-top: 1rem;
+}
+
+.Add_To_doList {
+  width: 100%;
+  padding: 0 40px 0 1rem;
+  line-height: 50px;
+  height: 50px;
+  border: none;
+  font-style: italic;
+}
+
+.Add_To_doList:focus,
+.list:focus {
+  outline: none;
+}
+
+.submit-ToDo {
+  display: flex;
+  justify-content: space-between;
+  padding-right: 1rem;
+  border-bottom: 1px solid hsl(0, 0%, 80%); /* Add a bottom border */
+}
+
+.card {
+  display: flex;
+  justify-content: space-between;
+  padding-left: 1rem;
+  padding-right: 1rem;
+}
+
+.h4-refresh {
+  display: flex;
+  justify-content: space-between;
+  padding-right: 1rem;
+  padding-top: 1rem;
+}
+
+hr {
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+}
+
+.clearSelected {
+  margin-top: 1rem;
+  text-align: center;
+  padding-top: 1.1rem;
+  padding-bottom: 1rem;
+  width: 100%;
+  border: none;
+  background-color: #eee;
+  font-size: 1.1rem;
+}
+
+.clearSelected:hover {
+  cursor: pointer;
+}
+
+nav {
+  width: 100%;
+  background-color: black;
+  height: 2rem;
+}
+
+footer {
+  width: 100%;
+  background-color: black;
+  height: 1rem;
+  position: fixed;
+  left: 0;
+  bottom: 0;
+}
+
+.box {
+  margin-right: 0.3rem;
+}
+
+.checked {
+  text-decoration: line-through;
+}
+
+.material-symbols-sharp:hover,
+.fa-rotate:hover {
+  cursor: pointer;
+}
+
+.more-sign {
+  margin-left: 10px;
+  cursor: pointer;
+}
+
+.todoList {
+  padding: 10px;
+}
+
+/* li {
```

Comment:
I suggest you could delete all commented-out codes and I think it could make the code looks clean. Leaving commented-out code in source code is bad practice, as it takes up space, causes confusion, and leads to maintenance issues if it’s not removed. Feel free to leave a comment.